protocol/package.json

```diff
@@ -14,6 +14,9 @@
 	},
 	"main": "./lib/main.js",
 	"typings": "./lib/main",
+	"devDependencies": {
```

Comment:
We shouldn't have this dependency here. Even not as dev. Best would be to have these tests in the cancellation package.

Reply:
The cancellation package only depends on jsonrpc for cancellation strategy interface. but looks like we don't care about custom cancellation strategy being tested on each layer where it is exposed to users as public API. so I can tests in protocol.